tests/DoctrineORMModuleTest/Yuml/YumlControllerFactoryTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace DoctrineORMModuleTest\Yuml;
+
+use DoctrineORMModule\Yuml\YumlController;
+use DoctrineORMModule\Yuml\YumlControllerFactory;
+use Zend\ServiceManager\AbstractPluginManager;
+use Zend\ServiceManager\ServiceLocatorInterface;
+
+class YumlControllerFactoryTest extends \PHPUnit_Framework_TestCase
+{
+    public function testCreateService()
+    {
+        $config = [
+            'zenddevelopertools' => [
+                'toolbar' => [
+                    'enabled' => true
+                ]
+            ]
+        ];
+
+        $serviceLocator = $this->getMockBuilder(ServiceLocatorInterface::class)
+            ->getMock();
+        $pluginManager = $this->getMockBuilder(AbstractPluginManager::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $serviceLocator->expects(static::any())->method('get')->with('config')->willReturn($config);
+
+        $pluginManager->expects(static::once())
+            ->method('getServiceLocator')
+            ->willReturn($serviceLocator);
+
+        $factory = new YumlControllerFactory();
+        $controller = $factory->createService($pluginManager);
+
+        static::assertInstanceOf(YumlController::class, $controller);
```

Comment:
@webimpress  
TestCase methods are instance methods:

https://github.com/sebastianbergmann/phpunit/blob/5.4.8/src/Framework/TestCase.php#L1509

Assert methods are static methods:

https://github.com/sebastianbergmann/phpunit/blob/5.4.8/src/Framework/Assert.php#L1320

Am I wrong using ```static::``` or it's a common mistake to use ```$this->``` for static methods? :)